src/main/java/com/google/flourbot/basic/Bot.java

```diff
@@ -54,128 +54,131 @@
 import java.security.GeneralSecurityException;
 import java.util.Collections;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 @SpringBootApplication
 @RestController
 public class Bot {
-    static final String CHAT_SCOPE = "https://www.googleapis.com/auth/chat.bot";
-    static final String SPREADSHEET_SCOPE = "https://www.googleapis.com/auth/spreadsheets";
+  static final String CHAT_SCOPE = "https://www.googleapis.com/auth/chat.bot";
+  static final String SPREADSHEET_SCOPE = "https://www.googleapis.com/auth/spreadsheets";
+
+  private static final Logger logger = Logger.getLogger(Bot.class.getName());
+
+  public static void main(String[] args) {
```

Comment:
yeah, main is a tricky subject, I'm not sure if we will need it, but if we do, not in this class... We can discuss it, but I think we should put it in the .flourbot package in a class that is only responsible for initialization.